client/src/components/PublicPages/AuditPlanner/ElectionResultsCard.tsx

```diff
@@ -15,16 +15,7 @@ import { StyledTable } from '../../Atoms/Table'
 import { sum } from '../../../utils/number'
```

Comment:
This commit is gonna be easier to review with [`?w=1`](https://github.com/votingworks/arlo/pull/1592/commits/28264e52b278ba589ff94a015a1636dba3f89dae?w=1)